model_signing/model.py

```diff
@@ -24,61 +24,68 @@
 )
 from sigstore_protobuf_specs.dev.sigstore.bundle.v1 import Bundle
 from sigstore.verify import (
-    VerificationMaterials,
     policy,
     Verifier,
 )
 from sigstore.verify.models import (
     VerificationMaterials,
-    VerificationResult,
```

Comment:
I think this one is still needed, used in 

```
def verify(self, inputfn: Path, signaturefn: Path,
               ignorepaths: [Path], offline: bool) -> VerificationResult:
```

Lines 130-152

Reply:
We are defining our own class here - https://github.com/google/model-transparency/pull/34/files#diff-5721baa8b2b4f803a185c5388ceecb5ff321e2d0a63671db95066063b32c3b78L109

I _think_ our redefinition is the correct one to use?

Reply:
Oh, I missed those lines, thank you for the clarification. In this case, this seems right to me.